products/green_fast_track/analyses/pluto_zoning_district_groups.sql

```diff
@@ -15,7 +15,7 @@ group_by_rollup AS (
         zonedist4,
         count(*) AS zoning_combo_count
     FROM pluto
-    GROUP BY ROLLUP (zonedist1, zonedist2, zonedist3, zonedist4) -- noqa: PRS
+    GROUP BY rollup(zonedist1, zonedist2, zonedist3, zonedist4) -- noqa: PRS
```

Comment:
nit but I think rollup is a keyword not a function so it should be uppercase

Reply:
yes! thank u, i forgot that sqlfluf is overzealous here